doc/manuals/user/v1_v2_coexistence.md

```diff
@@ -59,3 +59,21 @@ subscription. Apart from the values described in the NGSIv2 specification, Orion
 `legacy` value in order to send notifications in NGSIv1 format. This way, users can have the
 enhancements of NGSIv2 subscriptions (e.g. filtering) with NGSIv1 legacy notifications receivers.
 
+## NGSIv2 query update forwarding to Context Providers
+
+Context availability management functionality (i.e. operations to register Context Providers) is still to be
+implemented for NGSIv2. However, you can [register providers using NGSIv1 operations](context_providers.md)
+and have your NGSIv2-based updates and queries being forwarded to Context Providers, getting the response in NGSIv2.
+The forwarded message in the CB to CPr communication, and its response, is done using NGSIv1.
+
+However, the following considerations have to be taken into account:
+
+* Query filtering (e.g. `GET /v2/entities?q=temperature>40`) is not supported on query forwarding. First, Orion
+  doesn't include the filter in the `POST /v1/queryContext` operation forwarded to CPr. Second, Orion doesn't filter
+  the CPr results before responding them back to client. An issue corresponding to this limitation has been created:
+  https://github.com/telefonicaid/fiware-orion/issues/2282
+* On forwarding, any type of entity in the NGSIv2 update/query matches registrations without entity type. However, the
+  opposite doesn't work, so if you have registrations with types, then you must use `?type` in NGSIv2  update/query in
+  order to obtain a match.
+* In the case of partial updates (e.g. `POST /v2/op/entities` resulting in some entities/attributes being updated and
+  other entities/attributes not being updated due to failing or missing CPrs), 204 No Content is returned to the client.
```

Comment:
Note that "and information about all attributes not being updated is provided in the payload" has been removed. That's not true for release/1.2.0 (it is true in develop and in this documentation will be adapted in that branch to describe the right behavior).